js/piwik.js

```diff
@@ -4962,14 +5001,23 @@ if (typeof Piwik !== 'object') {
                 /**
                  * Set heartbeat (in seconds)
                  *
-                 * @param int minimumVisitLength
-                 * @param int heartBeatDelay
+                 * @param int heartBeatDelay Defaults to 15.
                  */
-                setHeartBeatTimer: function (minimumVisitLength, heartBeatDelay) {
-                    var now = new Date();
+                setHeartBeatTimer: function (heartBeatDelay) {
```

Comment:
It might break code that uses it, however, it didn't actually do anything before. It was incomplete, just half the feature. It sent useless requests, probably creating new actions, no one should be using it.